modules/test/jenkins-plugin-events/src/main/java/com/liferay/jenkins/plugin/events/JenkinsEventsRootAction.java

```diff
@@ -34,9 +105,15 @@ public String getIconFileName() {
 		return "clipboard.png";
 	}
 
+	public List<JenkinsWebHook> getJenkinsWebHooks() {
+		return jenkinsWebHooks;
+	}
+
 	@Override
 	public String getUrlName() {
 		return "jenkins-events";
 	}
 
+	public List<JenkinsWebHook> jenkinsWebHooks;
```

Comment:
@michaelhashimoto Is there a requirement for this to be public? Unless the Jenkins plugin API requires this to be public, I think this should be at least `protected` and `final` with a public getter method.